source/Halibut/HalibutRuntime.cs

```diff
@@ -338,6 +338,16 @@ public void Dispose()
 
         public async ValueTask DisposeAsync()
         {
+            foreach (var pendingRequestQueue in this.queues.Values)
```

Comment:
Oh what! We weren't disposing of these... Far out.

Just wondering (again), how did you notice this?

Reply:
Disregard. I see lower that disposing is a new thing.

Reply:
lol yes